search-service/src/main/resources/db/migration/V0_33__fix_instance_id_difference_in_its_column_and_payload.sql

```diff
@@ -0,0 +1,7 @@
+update attribute_instance
+set instance_id = trim('"' FROM jsonb_path_query_first(payload, '$."https://uri.etsi.org/ngsi-ld/instanceId"[0]."@id"')::text)
+where instance_id != trim('"' FROM jsonb_path_query_first(payload, '$."https://uri.etsi.org/ngsi-ld/instanceId"[0]."@id"')::text);
+
+update attribute_instance
+set instance_id = trim('"' FROM jsonb_path_query_first(payload, '$."instanceId"')::text)
+where instance_id != trim('"' FROM jsonb_path_query_first(payload, '$."instanceId"')::text);
```

Comment:
- the `trim` could be avoided by using such expression: `jsonb_path_query_first(payload, '$."https://uri.etsi.org/ngsi-ld/instanceId"[0]')->>'@id'`
- this script is going to break a lot of the values in the `instance_id` column